org/mozilla/jss/tests/TestGlobalReference.java

```diff
@@ -0,0 +1,23 @@
+package org.mozilla.jss.tests;
+
+import org.mozilla.jss.util.*;
+
+public class TestGlobalReference {
+    public static void main(String[] args) throws Exception {
+        String arg = "Something";
+
+        for (int i = 0; i < 100; i++) {
+            GlobalRefProxy proxy = new GlobalRefProxy(arg);
+            proxy.close();
+            proxy.close();
+            proxy.clear();
```

Comment:
Could you add a note describing what's being tested and the expected behavior? E.g. test calling `GlobalRefProxy.close()` multiple times, it should complete successfully.

Reply:
Added.